src/main/java/com/mongodb/jdbc/MongoBsonValue.java

```diff
@@ -34,14 +34,21 @@
  * does not extend BsonValue, instead it contains a BsonValue member.
  */
 public class MongoBsonValue {
-    static final JsonWriterSettings JSON_WRITER_SETTINGS =
-            JsonWriterSettings.builder().outputMode(JsonMode.RELAXED).build();
+    private JsonWriterSettings JSON_WRITER_SETTINGS;
     static final EncoderContext ENCODER_CONTEXT = EncoderContext.builder().build();
 
     private BsonValue v;
 
-    public MongoBsonValue(BsonValue v) {
+    public MongoBsonValue(BsonValue v, boolean isExt) {
```

Comment:
[nit] Just for better readability
```suggestion
    public MongoBsonValue(BsonValue v, boolean isExtended) {
```